helm/prometheus-meta-operator/templates/prometheus-rules/systemd.workload-cluster.rules.yml

```diff
@@ -0,0 +1,54 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+    cluster_type: "workload_cluster"
+  name: systemd.workload-cluster.rules
+spec:
+  groups:
+  - name: systemd
+    rules:
+    - alert: WorkloadClusterSystemdUnitFailed
+      annotations:
+        description: '{{`Systemd unit {{ $labels.name }} is failed on {{ $labels.instance }}.`}}'
+      expr: node_systemd_unit_state{cluster_type="workload_cluster",name!~"debug-tools.service|k8s-addons.service|sshd.+service|systemd-coredump.+service|systemd-networkd.service|update-engine-stub.service",state="failed"} == 1
```

Comment:
Removed `coreos-metadata-sshkeys@core.service`, and `locksmithd.service`. Is this correct @giantswarm/team-ludacris ?


original expr
```
node_systemd_unit_state{cluster_type="workload_cluster",name!~"coreos-metadata-sshkeys@core.service|debug-tools.service|k8s-addons.service|locksmithd.service|sshd.+service|systemd-coredump.+service|systemd-networkd.service|update-engine.service|update-engine-stub.service", state="failed"} == 1
```

Reply:
reverted to original expr